### PR TITLE
update ci setup

### PR DESCRIPTION
**Pull request by tomerd**
motivation: 5.2 adoption, prepare for 5.3

changes:
* use 5.2 docker-compose setup
* add 5.3 docker-compose setup (placeholder)
* update format

### CODE REVIEW

**Comment by tomerd** [comment]
@artemredkin @weissi 

```
Test Case 'HTTPClientTests.testRaceNewRequestsVsShutdown' started at 2020-04-04 02:15:14.707
Assertion failed: Invalid number of leased connections (-1): file /code/Sources/AsyncHTTPClient/ConnectionPool.swift, line 509
```

flaky test?

**Comment by artemredkin** [comment]
@tomerd yes, I'm working on it in #192

**Comment by artemredkin** [comment]
@swift-server-bot test this please

File: docker/docker-compose.1804.52.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    image: swift-metrics:18.04-5.2
+    build:
+      args:
+        ubuntu_version: "bionic"
+        swift_version: "5.2"
+
+  test:
+    image: swift-metrics:18.04-5.2
+    environment: []
+      #- SANITIZER_ARG=--sanitize=thread
```
Comment (artemredkin) [inline docker/docker-compose.1804.52.yaml:15]: why is sanitizer disabled?

File: docker/docker-compose.1804.52.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+version: "3"
+
+services:
+
+  runtime-setup:
+    image: swift-metrics:18.04-5.2
```
Comment (artemredkin) [inline docker/docker-compose.1804.52.yaml:6]: swift-metrics?

File: docker/docker-compose.1804.52.yaml (hunk 1 above)
Comment (tomerd) [reply docker/docker-compose.1804.52.yaml:15]: tsan issue on 18.04

**Comment by artemredkin** [comment]
@swift-server-bot test this please

**Comment by tomerd** [comment]
test mentioned ^^ is failing, i believe unrelated to this change

**Comment by artemredkin** [comment]
@swift-server-bot test this please

**Comment by artemredkin** [comment]
@swift-server-bot test this please

**Pull request merged**